x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/utils.test.ts

```diff
@@ -33,7 +33,7 @@ describe('generatePreviewUrl', () => {
         empty2: [''], // Empty fields should be stripped
       })
     ).toEqual(
-      'http://localhost:3002/as/engines/national-parks-demo/reference_application/preview?facets[]=baz&facets[]=qux&sortFields[]=quux&sortFields[]=quuz&titleField=foo&urlField=bar'
+      'http://localhost:3002/as/engines/national-parks-demo/search_experience/preview?facets[]=baz&facets[]=qux&sortFields[]=quux&sortFields[]=quuz&titleField=foo&urlField=bar'
```

Comment:
This is a new route that was set up specifically for this form, which accepts a search token in order to render the search experience preview, rather than authenticating like the former route.